rails/app/controllers/projects_controller.rb

```diff
@@ -3,7 +3,9 @@
 class ProjectsController < ApplicationController
   include ::ProjectsHelper
   before_action(except: %i[index create new]) { fetch_project(params[:id]) }
-  before_action(except: %i[show index create new reference_components]) { authorize(params[:project_object].editors) }
+  before_action(except: %i[show index create new reference_components changelog_viewed]) do |_action|
```

Comment:
this is fine for now, but it makes me think that we should re-work how the route this is called on.